src/bundle/Command/CronRunCommand.php

```diff
@@ -13,9 +13,24 @@
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
+use Psr\Log\LoggerInterface;
 
 class CronRunCommand extends ContainerAwareCommand
 {
+    /**
+     * @var \Psr\Log\LoggerInterface
+     */
+    private $logger;
+
+    /**
+     * @injectService $logger @logger
+     */
+    public function __construct(LoggerInterface $logger = null)
```

Comment:
Is this ever gonna be `null`? Dependency Injection configuration does not seem to expect that.

Reply:
@alongosz Fixed with new commit